build/resolver.go

```diff
@@ -0,0 +1,360 @@
+package build
```

Comment:
resolver as a filename in here is overused. In all codebases/pkgs it means a thing that provides a connection to image registry based on name.